README.md

```diff
@@ -1,27 +1,33 @@
-# Heroku Pack Base Images
+# Heroku Builder Images
 
 [![CircleCI](https://circleci.com/gh/heroku/pack-images.svg?style=svg)](https://circleci.com/gh/heroku/pack-images)
 
-This repository is responsible for building and publishing images that enable
-Heroku-like builds with [Cloud Native Buildpacks'](https://buildpacks.io)
-[`pack`](https://github.com/buildpacks/pack) command.
-
-| Image                                   | Base                                   | Type              | Status                                                                 |
-|-----------------------------------------|----------------------------------------|-------------------|------------------------------------------------------------------------|
-| [heroku/pack:18][pack-tags]             | [heroku/heroku:18][stack-images]       | CNB Run Image     | Deprecated (replaced by [heroku/heroku:18-cnb][heroku-tags])           |
-| [heroku/pack:18-build][pack-tags]       | [heroku/heroku:18-build][stack-images] | CNB Build Image   | Deprecated (replaced by [heroku/heroku:18-cnb-build][heroku-tags])     |
-| [heroku/buildpacks:18][buildpacks-tags] | [heroku/heroku:18-build][stack-images] | CNB Builder Image | Available                                                              |
-| [heroku/pack:20][pack-tags]             | [heroku/heroku:20][stack-images]       | CNB Run Image     | Deprecated (replaced by [heroku/heroku:20-cnb][heroku-tags])           |
-| [heroku/pack:20-build][pack-tags]       | [heroku/heroku:20-build][stack-images] | CNB Build Image   | Deprecated (replaced by [heroku/heroku:20-cnb-build][heroku-tags])     |
-| [heroku/buildpacks:20][buildpacks-tags] | [heroku/heroku:20-build][stack-images] | CNB Builder Image | Suggested                                                              |
-| [heroku/pack:22][pack-tags]             | [heroku/heroku:22][stack-images]       | CNB Run Image     | Deprecated (replaced by [heroku/heroku:22-cnb][heroku-tags])           |
-| [heroku/pack:22-build][pack-tags]       | [heroku/heroku:22-build][stack-images] | CNB Build Image   | Deprecated (replaced by [heroku/heroku:22-cnb-build][heroku-tags])           |
-| [heroku/buildpacks:22][buildpacks-tags] | [heroku/heroku:22-build][stack-images] | CNB Builder Image | In Development                                                         |
+This repository is responsible for building and publishing [Cloud Native Buildpacks](https://buildpacks.io)
+builders that enable Heroku-like builds with the [`pack`](https://github.com/buildpacks/pack) command.
+
+| Builder Image                                       | Base Image                        | Status         |
+|-----------------------------------------------------|-----------------------------------|----------------|
+| [`heroku/buildpacks:18`][buildpacks-tags]           | [`heroku/heroku:18`][heroku-tags] | Available      |
```

Comment:
Should these have the base images listed as the `-cnb-build` variants? (Though I guess technically they have two base images, the build and the runtime, but shrug :-) )

Reply:
Yeah, makes sense. Done.